.github/workflows/run_mmperf.yml

```diff
@@ -0,0 +1,106 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Workflow for running `mmperf` (https://github.com/mmperf/mmperf).
+# `mmperf` benchmarks matrix-multiply workloads on IREE and other backends such
+# as RUY, TVM, Halide, CuBLAS, etc.
+#
+# The workflow runs benchmarks on CPU and GPU and uploads results to the
```

Comment:
To be clear, it just runs IREE benchmarks, right?

Reply:
It runs benchmarks with IREE, SHARK-Runtime and various frameworks like TF and PyTorch. Here's an example of the output: https://storage.cloud.google.com/mmperf-benchmark-artifacts/cpu/2022-11-03_00-23-03-971824/matmul.png